### PR TITLE
Updated tests for schedule veteran form and added tests for schedule …

### PR DESCRIPTION
**Pull request by sahalliburton**
Tests for the PR to split out the ScheduleVeteran wrapper

### Description
Updates and adds jest tests for the ScheduleVeteran workflow

### Acceptance Criteria
- [ ] Code compiles correctly

### Testing Plan
1. Run jest tests locally `yarn run test:jest -u`
1. Ensure all tests pass

### Storybook Story
*For Frontend (Presentationa) Components*
* [x] Add a [Storybook](https://github.com/department-of-veterans-affairs/caseflow/wiki/Documenting-React-Components-with-Storybook) file alongside the component file (e.g. create `MyComponent.stories.js` alongside `MyComponent.jsx`)
* [x] Give it a title that reflects the component's location within the overall Caseflow hierarchy 
* [x] Write a separate story (within the same file) for each discrete variation of the component

### CODE REVIEW

File: client/app/hearings/components/ScheduleVeteran.jsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const ScheduleVeteran = ({
 
   // Determine the Request Type for the hearing
   const virtual = assignHearingForm?.virtualHearing;
-  const requestType = HEARING_REQUEST_TYPES[selectedHearingDay.requestType] || 'Video';
+  const requestType = HEARING_REQUEST_TYPES[selectedHearingDay?.requestType] || 'Video';
```
Comment (sahalliburton) [inline client/app/hearings/components/ScheduleVeteran.jsx:59]: Added safe-assign to fix errors when missing

File: client/test/app/hearings/components/ScheduleVeteranForm.test.js [hunk 2]
```diff
@@ -75,109 +70,41 @@ describe('ScheduleVeteranForm', () => {
     expect(scheduleVeteran).toMatchSnapshot();
   });
 
-  test('Can cancel the form', () => {
```
Comment (sahalliburton) [inline client/test/app/hearings/components/ScheduleVeteranForm.test.js:78]: Moved these tests into the ScheduleVeteran component tests

File: client/test/app/hearings/components/ScheduleVeteranForm.test.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { shallow } from 'enzyme';
 
 import { ScheduleVeteranForm } from 'app/hearings/components/ScheduleVeteranForm';
 import { ReadOnly } from 'app/hearings/components/details/ReadOnly';
-import { amaAppeal, defaultHearing } from 'test/data';
+import { amaAppeal, defaultHearing, virtualHearing } from 'test/data';
 import { queueWrapper, queueStore } from 'test/data/stores/queueStore';
```
Comment (tomas-nava) [inline client/test/app/hearings/components/ScheduleVeteranForm.test.js:7]: I don't think `queueStore` is being used anymore (not sure why lint didn't catch this).

Comment (sahalliburton) [reply client/test/app/hearings/components/ScheduleVeteranForm.test.js:7]: Thanks I think you are right and I do not believe the lint is setup to catch storybook stories for some reason

File: client/app/hearings/components/ScheduleVeteran.jsx [hunk 4]
```diff
@@ -142,10 +140,8 @@ export const ScheduleVeteran = ({
         regionalOffice: hearing.regionalOffice ? null : 'Please select a Regional Office '
       };
 
-      console.log(formErrors);
-
       // First validate the form
-      if (openHearing || formErrors) {
+      if (openHearing || Object.keys(formErrors).filter((err) => formErrors[err] !== null).length > 0) {
```
[the remark this answers is not in the document]
Comment (sahalliburton) [reply client/app/hearings/components/ScheduleVeteran.jsx:144]: Yeah you are absolutely right that would definitely be much more concise, using `Object.keys` is an old habit I need to break. Thanks!

**Pull request merged**